plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -63,11 +63,6 @@ export const PricingSection: React.FC = () => {
 		return cleanValue;
 	};
 
-	const taxSettingsText =
-		'Per your {{link}}store settings{{/link}}, tax is {{strong}}%sincluded{{/strong}} in the price.';
-	const addNot = pricesIncludeTax ? '' : 'not ';
-	taxSettingsText.replace( taxSettingsText, addNot );
-
```

Comment:
We will need the `addNot` to add it when the tax is not included in the price. 
The code should look like this:
```
	const addNot = pricesIncludeTax ? '' : 'not ';
	const taxSettingsElement = interpolateComponents( {
		mixedString: sprintf(
			__(
				'Per your {{link}}store settings{{/link}}, tax is {{strong}}%sincluded{{/strong}} in the price.',
				'woocommerce'
			),
			addNot
		),
		components: {
			link: (
				<Link
					href={ `${ ADMIN_URL }admin.php?page=wc-settings&tab=tax` }
					target="_blank"
					type="external"
					onClick={ () => {
						recordEvent( 'product_pricing_list_price_help' );
					} }
				>
					<></>
				</Link>
			),
			strong: <strong />,
		},
	} );
```

I missed that in my original PR.